src/databricks/labs/ucx/install.py

```diff
@@ -513,7 +517,7 @@ def warehouse_type(_):
             instance_profile=instance_profile,
             spark_conf=spark_conf_dict,
             override_clusters=self._install_override_clusters,
-            custom_cluster_policy_id=custom_cluster_policy_id,
+            custom_cluster_policy_id=self._custom_cluster_policy_id,
```

Comment:
just introduce the `policy_id` field, that would be always required now. copy the rules from this `_custom_cluster_policy_id` and don't use it anymore. Pay special attention to AWS Glue and external hive metastores.